stabilizer/src/main/java/com/hazelcast/stabilizer/coordinator/Coordinator.java

```diff
@@ -52,6 +52,7 @@
 public class Coordinator {
 
     public final static File STABILIZER_HOME = getStablizerHome();
+    public final static String UPLOAD_HOME = STABILIZER_HOME.getAbsolutePath() + "/upload";
```

Comment:
You are still in the wrong directory.

You do not want to have an upload directory in the stabilizer, but you want to have an upload directory in your 'working directory'. Please contact me.